whelktool/scripts/2023/02/lxl-1556-version-vs-marc-version/2-marc-version-to-version.groovy

```diff
@@ -19,7 +19,8 @@ MARC_VERSION_BIB_PATHS =
                 ['instanceOf', 'expressionOf'],
                 ['instanceOf', 'hasPart'],
                 ['instanceOf', 'relationship', '*', 'entity'],
-                ['instanceOf', 'subject']
+                ['instanceOf', 'subject'],
+                ['instanceOf', 'seriesMembership', 'inSeries', 'instanceOf']
```

Comment:
```suggestion
                ['seriesMembership', '*', 'inSeries', 'instanceOf']
```

Reply:
`seriesMembership` is on the instance right? Since `seriesMembership` is an array we also need the asterisk to get all objects from that array.